### DISCUSSION TITLE
Mock URL for the component

### DISCUSSION THREAD

**Discussion by shorstki** [storybookjs/storybook#24752] [Ideas]
### Is your feature request related to a problem? Please describe.

I have component (Angular) that behaves different, depending on the current URL. Is there a way to set the URL to be visible for the displayed component, like it'd appear in real browser?

### Describe the solution you'd like

Param in config to mock URL for the presented component (and child components)

### Describe alternatives you've considered

No alternatives for Angular, could be done only by some E2E/unit tests

### Are you able to assist to bring the feature to reality?

no

### Additional context

There is plugin for React router, but nothing for Angular

**Comment by Marklb** [comment]
This seems like a feature for an addon to provide. I am not positive what you are wanting, based on the description. Are you wanting to display the URL or set the URL? Have you looked at [storybook-addon-angular-router](https://storybook.js.org/addons/storybook-addon-angular-router)?

You said there is a plugin for React, but did not mention which one you are referring to. Pointing to the React solution could help with knowing what you are wanting for Angular.
